composer.json

```diff
@@ -28,5 +28,10 @@
         "psr-4": {
             "AppTest\\": "test/AppTest/"
         }
+    },
+    "scripts": {
+        "phpcs": "phpcs",
+        "server": "php -S localhost:8080 -t public",
```

Comment:
Unfortunately you have to run this with `composer run-script server --timeout 0` so that the command doesn't timeout